containers/Ideas/Idea.js

```diff
@@ -1,62 +1,83 @@
 import React from 'react';
-import { View, Text, Image } from 'react-native';
+import { View, Text, Image, ScrollView, StatusBar } from 'react-native';
 import { Button } from 'react-native-elements';
 import { styles } from './Idea.styles';
 import Icon from 'react-native-vector-icons/SimpleLineIcons';
 
-import { ButtonCounter } from '../../components/buttonCounter';
+import { ButtonCounter } from '../../components/ButtonCounter';
```

Comment:
This gives an error because the files in components are not yet renamed. 
```
Failed building JavaScript bundle.
Unable to resolve module ../../components/ButtonCounter from /home/katharina/zt-app/containers/Ideas/Idea.js: 

None of these files exist:
  * components/ButtonCounter(.native|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx|.android.js|.native.js|.js|.android.jsx|.native.jsx|.jsx|.android.json|.native.json|.json)
  * components/ButtonCounter/index(.native|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx|.android.js|.native.js|.js|.android.jsx|.native.jsx|.jsx|.android.json|.native.json|.json)
   5 | import Icon from 'react-native-vector-icons/SimpleLineIcons';
   6 |
>  7 | import { ButtonCounter } from '../../components/ButtonCounter';
     |                                ^
   8 | import { Chip } from '../../components/Chip';
   9 |
  10 | export const Idea = (props) => {
```